### PR TITLE
Staff dg 1

### PR DESCRIPTION
**Pull request by Lim-Shi-Han**
-Create sequence diagrams for add and delete staff
-Change method (isValidName() in Staff and Dish) as mentioned in issue
-Handle all print statements for Staff classes in StaffController

### CODE REVIEW

File: logs/application.log [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (darylhjd) [inline logs/application.log:1]: did you fetch from upstream? dont commit logs, and the gitignore should also include logs folder once you fetch.

**Review by darylhjd — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/duke/entities/Dish.java [hunk 2]
```diff
@@ -29,7 +29,7 @@ private static String toTitleCase(String givenString) {
     }
 
     private static boolean isValidName(String name) {
-        return !(name == null || name.equals(""));
+        return !(name == null || name.length() == 0);
```
Comment (darylhjd) [inline src/main/java/seedu/duke/entities/Dish.java:32]: you do know theres a `isEmpty` method or smth right lol

Comment (darylhjd) [reply src/main/java/seedu/duke/entities/Dish.java:32]: also im not sure how name would ever be null if you use getString method of InputParser

Comment (darylhjd) [reply src/main/java/seedu/duke/entities/Dish.java:32]: so technically, you can just check for name.isEmpty() in the constructor and throw IllegalArgumentException there, dunnid a extra method to check valid name.

Comment (Lim-Shi-Han) [reply src/main/java/seedu/duke/entities/Dish.java:32]: Okay I go change

File: logs/application.log (hunk 1 above)
Comment (Lim-Shi-Han) [reply logs/application.log]: Will handle it in another PR, as we will have to change the JUnit tests also

File: src/main/java/seedu/duke/entities/Dish.java (hunk 2 above)
Comment (Lim-Shi-Han) [reply src/main/java/seedu/duke/entities/Dish.java:32]: Will handle it in another PR, as we will have to change the JUnit tests also

Comment (darylhjd) [reply src/main/java/seedu/duke/entities/Dish.java:32]: ok ill leave it to u

**Review by darylhjd — APPROVED** [review]

File: src/main/java/seedu/duke/entities/Dish.java (hunk 2 above)
Comment (BotBw) [reply src/main/java/seedu/duke/entities/Dish.java:32]: > so technically, you can just check for name.isEmpty() in the constructor and throw IllegalArgumentException there, dunnid a extra method to check valid name.

still need the boolean there cuz I used `isValidName` for some assertion, maybe only change to `name.isEmpty()`?

Comment (Lim-Shi-Han) [reply src/main/java/seedu/duke/entities/Dish.java:32]: > > so technically, you can just check for name.isEmpty() in the constructor and throw IllegalArgumentException there, dunnid a extra method to check valid name.
> 
> still need the boolean there cuz I used `isValidName` for some assertion, maybe only change to `name.isEmpty()`?

I think we can still use name.isEmpty() for assertions, but then the JUnit tests I encountered some problems with our tests so might have to change that part

But assertions shld be fine by using `assert !(name.isEmpty())`

Comment (BotBw) [reply src/main/java/seedu/duke/entities/Dish.java:32]: > > > so technically, you can just check for name.isEmpty() in the constructor and throw IllegalArgumentException there, dunnid a extra method to check valid name.
> > 
> > 
> > still need the boolean there cuz I used `isValidName` for some assertion, maybe only change to `name.isEmpty()`?
> 
> I think we can still use name.isEmpty() for assertions, but then the JUnit tests I encountered some problems with our tests so might have to change that part
> 
> But assertions shld be fine by using `assert !(name.isEmpty())`

I guess that's I tested null value haha (i ll delete that later), then it's fine.

**Pull request merged**